public/app/features/browse-dashboards/components/TagsCell.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { CellProps } from 'react-table';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { TagList, useStyles2 } from '@grafana/ui';
+
+import { DashboardsTreeItem } from '../types';
+
+export function TagsCell({ row: { original: data } }: CellProps<DashboardsTreeItem, unknown>) {
+  const styles = useStyles2(getStyles);
+  const item = data.item;
+  if (item.kind === 'ui-empty-folder') {
+    return <></>;
```

Comment:
minor nit: is there a benefit to returning an empty fragment instead of `null`? 🤔

Reply:
typescript disallows it. I _think_ Typescript 5.1 might fix this.